src/ui.tsx

```diff
@@ -589,6 +589,39 @@ function PlanStatistics({plan}: {plan: Plan}) {
   </div>;
 }
 
+function TimeLeft({plan, progress, currentLineStartedTime, paused}: {
+  plan: Plan;
+  progress: number | null; 
+  currentLineStartedTime: Date | null;
+  paused: boolean;
+}) {
+  const [_, setTime] = useState(new Date());
+
+  // Interval that ticks every second to rerender
+  // and recalculate time remaining for long lines
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTime(new Date());
+    }, 1000);
+
+    () => {
+      clearInterval(interval);
+    }
+  }, [setTime])
+
+  if (!plan || !plan.duration || progress === null || paused) {
+    return null;
+  }
+
+  const currentLineProgress = (new Date().getTime() - currentLineStartedTime.getTime()) / 1000;
+  const duration = plan.duration(progress);
+
+  return <div className="duration">
+    <div className="time-remaining-label">Time remaining:</div>
```

Comment:
```suggestion
    <div className="time-remaining-label">Time remaining</div>
```

To match the `Duration` label, which doesn't have a `:`